packages/react-hooks/src/useId.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { getId } from '@fluentui/utilities/lib/getId';
+import { getId } from '@fluentui/utilities';
```

Comment:
This one may have been intentional to avoid importing/parsing any more than necessary? But it's hard to say whether that was the case or if it still matters since there's no comment.

Reply:
I'm pretty convinced it saves little to no parse time or adds side effects. If we think it is adding anything significant, this could (should?) be moved to it's own package. Probably marked as a singleton package since I think it retains file scope state.

Reply:
I saw that because likely if you're using react-hooks, you're probably also using utilities.

Reply:
Yeah I don't *think* there should be any significant issues? Current me is not impressed with past me for not documenting intentions here better. :P